packages/cbioportal-utils/src/hotspot/CancerHotspotsUtils.ts

```diff
@@ -106,6 +106,15 @@ export function filter3dHotspotsByMutations(
     );
 }
 
+export function filterSpliceHotspotsByMutations(
+    mutations: Mutation[],
+    index: IHotspotIndex
+): Hotspot[] {
+    return filterHotspotsByMutations(mutations, index, (hotspot: Hotspot) =>
+        hotspot.type.toLowerCase().includes('splice')
```

Comment:
We may also consider adding this condition into `filterRecurrentHotspotsByMutations`. Is there any specific reason that we may want to treat splice hotspots as a separate category? We needed to do this for 3d hotspots for visualization purposes, but not sure if this is necessary for splice mutations.

Reply:
because only splice mutations should match splice hotspots as we discussed